ts/components/session/SessionSeedModal.tsx

```diff
@@ -40,6 +40,37 @@ export class SessionSeedModal extends React.Component<Props, State> {
     window.addEventListener('keyup', this.onEnter);
   }
 
+  public render() {
+    const i18n = window.i18n;
+
+    this.checkHasPassword();
+    this.getSeed().ignore();
```

Comment:
Those are calls should rather be in the `componentWillMount()` method. 

If we keep it there, it can cause a render-loop. If for some reason `seed` is set to an empty string in the `getSeed()`, the `setState()` will recall `render()` which will loop again and again.